changedetectionio/store.py

```diff
@@ -578,8 +582,28 @@ def update_5(self):
 
     # We incorrectly used common header overrides that should only apply to Requests
     # These are now handled in content_fetcher::html_requests and shouldnt be passed to Playwright/Selenium
-    def update_7(self):
+    def update_6(self):
```

Comment:
oops sorry, I made a bad conflict resolve, could you change this to

```suggestion
    def update_8(self):
```